Neptune/IMessage.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Neptune
+{
+
+    public interface IRequest<IS>
+        where IS : IService
+    { }
+
+    public interface IGetRequest<IS> : IRequest<IS>
```

Comment:
I'm actually not a fan of these shortened names. It makes code significantly less readable.

Reply:
Do you mean the interface itself, or the generic?

Reply:
Let’s stick with long names. We have auto complete anyways

Reply:
The generic